pos-print/src/main/java/com/clouway/pos/print/printer/FP705Printer.java

```
@@ -160,7 +160,7 @@ public void reportForOperator(String operatorId, RegisterState state) throws IOE
 
   @Override
   public PrintReceiptResponse printFiscalReceipt(Receipt receipt) throws IOException {
-    WarningChannel channel = createChannel(5);
+    WarningChannel channel = createChannel(50);
```

Removed the default value since it was misleading.